lib/homePage.dart

```diff
@@ -90,17 +90,28 @@ class CategoryCard extends StatelessWidget {
           onTap: onPressed,
           child: Container(
               decoration: BoxDecoration(
+                  borderRadius: BorderRadius.circular(30),
```

Comment:
This circular Container looks amazing, but the whole application is based on a rectangular shape. So in this application Ragtengle would be better! what do you think?